webhonc/src/index.ts

```diff
@@ -0,0 +1,94 @@
+import { zValidator } from "@hono/zod-validator";
+import { Hono } from "hono";
+import { z } from "zod";
+import type { Bindings, Variables } from "./types";
+import { WebHonc } from "./webhonc";
+
+const app = new Hono<{ Bindings: Bindings; Variables: Variables }>();
+
+app.get("/ws", async (c) => {
+  if (c.req.header("upgrade") !== "websocket") {
+    return new Response("Not a websocket request", { status: 426 });
+  }
+
+  const id = c.env.WEBHONC.newUniqueId();
+  const webhonc = c.env.WEBHONC.get(id) as DurableObjectStub<WebHonc>;
```

Comment:
TODO - handle the case where nothing is returned? right now if i pass in a random id to this route, we just get an internal server error. a 404 could be nicer

Reply:
here it shouldn't be a problem because id is created using DO's `newUniqueId()`, however, in the other route it is indeed a problem